src/actions/track-chair-actions.js

```diff
@@ -71,10 +72,13 @@ export const getTrackChairs =
     }
 
     const params = {
-      page: page,
+      page,
       per_page: perPage,
       access_token: accessToken,
-      expand: "member,categories"
+      expand: "member,categories",
+      relations: "member.none,categories.none",
+      fields:
+        "categories.id,categories.name,member.first_name,member.last_name,member.email"
```

Comment:
i dont think that u need category.id
also created, last_edited and summit_id could be removed from main entity

Reply:
correct fields are
     "id,categories.id,categories.name,member.first_name,member.last_name,member.email"